engine/src/gfx/cockpit.cpp

```diff
@@ -1944,20 +1944,20 @@ void GameCockpit::Draw() {
 }
 
 int GameCockpit::getScrollOffset(unsigned int whichtype) {
-    for (unsigned int i = 0; i < vdu.size(); i++) {
-        if (vdu[i]->getMode() & whichtype) {
-            return vdu[i]->scrolloffset;
+    for (std::size_t i = 0; i < vdu.size(); ++i) {
```

Comment:
Why not go full modern with 

```
for(auto& vdu_unit : vdu) {
...

Note that `vdu` really should be `vdus`.
```

Reply:
I could probably do that as well. I'm a bit lazy a lot of the time, and only go "full modern," as you put it, when CLion gives me an easy shortcut to click to accomplish that.